integration/package.json

```
@@ -10,24 +10,24 @@
     "dev": "lerna run test --scope integration --parallel --include-filtered-dependencies"
   },
   "dependencies": {
-    "@shapeshiftoss/hdwallet-core": "1.50.4",
-    "@shapeshiftoss/hdwallet-keepkey": "1.50.4",
-    "@shapeshiftoss/hdwallet-keepkey-nodewebusb": "1.50.4",
-    "@shapeshiftoss/hdwallet-keepkey-tcp": "1.50.4",
-    "@shapeshiftoss/hdwallet-ledger": "1.50.4",
-    "@shapeshiftoss/hdwallet-native": "1.50.4",
-    "@shapeshiftoss/hdwallet-portis": "1.50.4",
-    "@shapeshiftoss/hdwallet-trezor": "1.50.4",
-    "@shapeshiftoss/hdwallet-xdefi": "1.50.4",
+    "@shapeshiftoss/hdwallet-core": "^1.50.5-alpha.68",
+    "@shapeshiftoss/hdwallet-keepkey": "^1.50.5-alpha.68",
+    "@shapeshiftoss/hdwallet-keepkey-nodewebusb": "^1.50.5-alpha.68",
+    "@shapeshiftoss/hdwallet-keepkey-tcp": "^1.50.5-alpha.68",
+    "@shapeshiftoss/hdwallet-ledger": "^1.50.5-alpha.68",
+    "@shapeshiftoss/hdwallet-native": "^1.50.4",
+    "@shapeshiftoss/hdwallet-portis": "^1.50.4",
```

everything should be pinned to same version and regen lock file
```suggestion
    "@shapeshiftoss/hdwallet-native": "^1.50.4-alpha.68",
    "@shapeshiftoss/hdwallet-portis": "^1.50.4-alpha.68",
```

---

looks like lerna is on `alpha.72` actually. all packages should be fixed to the same version if using lerna

---

going to push an update to change that and also bump `metamask-snaps-adapter` to the latest alpha version